### DISCUSSION TITLE
Lock entry from edits when someone is editing the entry

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4593] [Feature Requests]
### Suggest an idea for ExpressionEngine

I’d love to see a feature in ExpressionEngine that helps prevent version conflicts when multiple people try editing the same entry at once. Here’s the idea:

1. Alert: Show a message if someone else is already editing the entry, maybe even their name (like “John Doe is editing this entry”).
2. Lock: Lock the entry so others can’t edit it until the first person is done: make fields and buttons inactive
3. Auto-Unlock: If someone leaves the page (or is inactive for too long), unlock it automatically, updating its state to the last saved version.

**Comment by swierczek** [comment]
Noting that the [Too Many Cooks](https://expressionengine.com/add-ons/too-many-cooks) add-on handles this, but a native solution would be great.

**Comment by intoeetive** [comment]
Not exactly what's being requested, but at least adding some information about possible interferation of other user: https://github.com/ExpressionEngine/ExpressionEngine/pull/4864
